packages/ui/src/components/vuestic-components/va-switch/VaSwitch.vue

```diff
@@ -1,26 +1,21 @@
 <template>
   <div
-    class="va-toggle"
+    class="va-switch"
     :class="computedClass"
-    @click="toggle"
-    @keydown.enter="toggle"
+    @click="switchValue"
+    @keydown.enter="switchValue"
     @mousedown="hasMouseDown = true"
     @mouseup="hasMouseDown = false"
     :tabindex="computedTabindex"
-    @focus="onFocus"
     @blur="isKeyboardFocused = false"
   >
-    <div class="va-toggle__inner">
-      <span
-        class="va-toggle__track"
-        :style="trackStyle"
-      />
-      <span
-        class="va-toggle__input"
-        :style="indicatorStyle"
-      />
+    <div class="va-switch__inner">
+      <span class="va-switch__track" :style="trackStyle" />
+      <span class="va-switch__input" :style="indicatorStyle">
+        <va-progress-circle v-if="loading" indeterminate :size="computedProgressCircleSize" :color="color || 'black'" />
```

Comment:
we probably want to use `getColor` method from color mixin. It gets theme support.